assets/frontend/js/globalpayments-applepay.js

```diff
@@ -61,7 +61,8 @@
 
 			var self = this
 			var paymentButton = document.createElement( 'div' );
-			paymentButton.className = 'apple-pay-button apple-pay-button-white-with-line';
+
+			paymentButton.className = 'apple-pay-button' + ' apple-pay-button-' + this.gatewayOptions.button_color;
```

Comment:
I don't think there is any reason to concatenate the first two strings, we can have them as 'apple-pay-button apple-pay-button-'